products/salt.md

```diff
@@ -0,0 +1,67 @@
+---
+
+title: Salt
+category: server-app
+tags: salt
+iconSlug: saltproject
+permalink: /salt
+versionCommand: salt --version
+releasePolicyLink: https://docs.saltproject.io/salt/install-guide/en/latest/topics/salt-version-support-lifecycle.html
+releaseImage: https://docs.saltproject.io/salt/install-guide/en/latest/_images/salt-release-timeline.png
+changelogTemplate: https://docs.saltproject.io/en/latest/topics/releases/__LATEST__.html#changelog
+eolColumn: CVE & Critical Support
+activeSupportColumn: Active Support
+releaseColumn: true
+releaseDateColumn: true
+
+auto:
+  - git: https://github.com/saltstack/salt.git
+
+identifiers:
+  - repology: salt
+
+releases:
+
+-   releaseCycle: "3006"
+    releaseDate: 2023-04-18
+    support: 2024-04-18
+    eol: 2025-04-18
+    latest: "3006.1"
+    latestReleaseDate: 2023-05-05
+    lts: true
+    link: https://docs.saltproject.io/en/latest/topics/releases/3006.0.html#changelog
+
+-   releaseCycle: "3005"
+    releaseDate: 2022-08-25
+    support: 2023-08-25
+    eol: 2024-02-25
+    latest: "3005.1"
+    latestReleaseDate: 2022-11-08
+    link: https://docs.saltproject.io/en/latest/topics/releases/3005.1.html
+
+-   releaseCycle: "3004"
+    releaseDate: 2021-10-18
+    support: 2022-10-18
+    eol: 2023-04-18
+    latest: "3004.2"
+    latestReleaseDate: 2022-06-21
+    link: https://docs.saltproject.io/en/latest/topics/releases/3004.html
+
+---
+
+> [Salt](https://saltproject.io/index.html) is software to automate the management and configuration of any infrastructure or application at scale.
+
+
+## Support Lifecycle
+Beginning with the 3006 release of Salt, the Salt Project will follow an LTS/STS release strategy. Under this strategy, the Salt Project will release one LTS (long-term support) release of Salt per year and one STS (short-term support) release each year. The LTS release will receive bug fix releases for a longer period of time than STS releases.
+
+The purpose of the LTS release is to provide users with a stable version of Salt for a longer period (2 years). The purpose of the STS release is to provide a feature release to users who need or prefer access to the latest features in between LTS releases.
+
+## Support Comparison
+
+|                      |                                                                                                                                LTS release                                                                                                                               |                                                                                                   STS release                                                                                                   |
```

Comment:
I find tables hard to maintain, is it okay if I were to split this into separate sections instead (Support, Release Cadence)

Reply:
I think we can drop the table all together, it is documented on the salt website.